pom.xml

```diff
@@ -101,6 +78,107 @@
+                <module>geomesa-convert</module>
+                <module>geomesa-tools-common</module>
+                <module>geomesa-web</module>
+                <module>geomesa-process</module>
+                <module>geomesa-stream</module>
+                <module>geomesa-z3</module>
+                <module>geomesa-accumulo</module>
+                <module>geomesa-hbase</module>
+                <module>geomesa-blobstore</module>
+                <module>geomesa-cassandra</module>
+                <module>geomesa-metrics</module>
+                <module>docs</module>
+                <module>geomesa-native-api</module>
+                <module>geomesa-index-api</module>
+            </modules>
+        </profile>
+
+        <profile>
+            <id>scala-2.11</id>
+            <activation>
+                <property><name>!scala-2.10</name></property>
+            </activation>
+            <properties>
+                <scala.version>2.11.7</scala.version>
+                <scala.binary.version>2.11</scala.binary.version>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.typesafe.scala-logging</groupId>
+                        <artifactId>scala-logging_${scala.binary.version}</artifactId>
+                        <version>${scalalogging.version}</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.scala-lang.modules</groupId>
+                        <artifactId>scala-parser-combinators_${scala.binary.version}</artifactId>
+                        <version>${scala.parsers.version}</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.scala-lang.modules</groupId>
+                        <artifactId>scala-xml_${scala.binary.version}</artifactId>
+                        <version>${scala.parsers.version}</version>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
+        </profile>
+
+        <profile>
+            <id>scala-2.10</id>
+            <activation>
+                <property><name>scala-2.10</name></property>
+            </activation>
+            <properties>
+                <scala.version>2.10.6</scala.version>
+                <scala.binary.version>2.10</scala.binary.version>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.typesafe.scala-logging</groupId>
+                        <artifactId>scala-logging-slf4j_${scala.binary.version}</artifactId>
+                        <version>2.1.2</version>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
+        </profile>
+
+        <profile>
+            <id>spark</id>
```

Comment:
Not necessarily.  Since this back port was originally done for Spark I constrained the 2.10 source back porting only the modules needed for spark.  This was captured in a profile so that code could be compiled.